pkgs/by-name/wl/wlvncc/package.nix

```diff
@@ -24,13 +24,13 @@
 }:
 stdenv.mkDerivation {
   pname = "wlvncc";
-  version = "unstable-2023-01-05";
+  version = "unstable-2024-11-23";
```

Comment:
Nit: Should be `0-unstable-2024-11-23`, see https://github.com/NixOS/nixpkgs/blob/522d5d85db83d3c9960edbedf1ba5a1b2001707d/pkgs/README.md#versioning .